CONTRIBUTING.md

```diff
@@ -1,28 +1,100 @@
 # Contributing to the AllenSDK
 
-Thank you for your interest in contributing!
+Thank you for your interest in contributing! There are a few ways you can contribute
+to the AllenSDK project:
 
-### Bug reports and feature requests
+* Answering questions from users asking for help on the [forums](https://community.brain-map.org/tag/allensdk) or on [StackOverflow](https://stackoverflow.com/questions/tagged/allen-sdk)
+* Adding test coverage
+* Updating release documentation
+* Reporting bugs
+* Requesting features
+* Contributing code
 
-* Before reporting a bug or requesting a feature, use Github's issue search to see if anyone else has already done so.
-* If there is no existing issue, create a new one. You should include:
-    * A brief, descriptive title
-    * A clear description of the problem or request.
-    * If you are reporting a bug, your description should contain the following information:
-        * What you did (preferably the actual code or commands that you ran)
-        * What happened
-        * What you were expecting to happen
-        * How your system is configured (operating system, Python version)
-* If you are comfortable addressing this issue yourself, take a look at the [guide to contributing code](#code) below.
+## Table of Contents
+* [Answering User Questions](##Answering-UserQuestions)
+* [Bug Reports and Feature Requests](##Bug-Reports-and-Feature-Requests)
+    * [Reporting Bugs](###Reporting-Bugs)
+    * [Suggesting Features/Enhancements](###Suggesting-Features/Enhancements)
+    * [Asking Questions](###Asking-Questions)
+* [Contributing Code](##Contributing-Code)
+    * [Deciding What to Contribute](###Deciding-What-to-Contribute)
+    * [Setting Up](###Setting-Up)
+    * [Style Guidelines](###Style-Guidelines)
+        * [Docstrings](####Docstrings)
+    * [Testing Guidelines](###Testing-Guidelines)
+    * [Committing Guidelines](###Committing-Guidelines)
+    * [Making a Pull Request](###Making-a-pull-request)
+    * [Review Process](###Review-process)
 
-### Questions 
+
+## Answering User Questions
```

Comment:
I disagree. We have no idea if contributors are "less than knowledgeable." Assuming good intentions, if users feel comfortable publicly answering questions about the AllenSDK, then they probably know how to use it. If they are wrong, then the code won't work or another user will correct it. That's the nature of open source. We already have open forums and a StackOverflow tag, so this isn't different than the status quo (other than pointing users to it).

Reply:
Yes I agree that generally that works but we don't have guarantees that anyone else will answer or an upvote downvote system like StackOverflow, so correct me if I'm wrong but we have the potential to have answers put up publicly with no secondary information if no one sees it to correct it. Our StackOverflow tagged posts for example most have one answer or none and hardly have upvotes. I think that the state of the StackOverflow posts is good because we have an Allen dev answering the majority of the posts. Having non experts can lead to inaccurate or out of date information but if that's the direction that is desired that's fine.

Reply:
It's definitely reasonable for you to be concerned about the quality of information provided -- of course we want our product to be represented as accurately as possible. One reason I'm not so worried about inaccurate statements from community members is because the credibility of the person answering matters. 

Ultimately I think this is going to come down to philosophical differences. I value the freedom of users, and I don't think we should discourage taking part in community discussions. If we see that people are getting stuck on the forums we can always jump in -- as AllenSDK team members we have higher credibility with respect to this tool than non-committers.

Reply:
I'm fine with that as the direction forward, I have trepidation that some things will slip as we move further from this PR but that's a bridge to come to when it occurs.